services/shhext/chat/encryption.go

```diff
@@ -251,6 +251,7 @@ func (s *EncryptionService) createNewSession(drInfo *RatchetInfo, sk [32]byte, k
 			keyPair,
 			s.persistence.GetSessionStorage(),
 			dr.WithKeysStorage(s.persistence.GetKeysStorage()),
+			dr.WithMaxSkip(10000),
```

Comment:
Add comment maybe with an explanation that it's a temporary or link to this PR description?

Reply:
@cammellos I would mark it as `TODO` or `FIXME` here

Reply:
done!